testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/actions/AbstractRequiredActionUpdateEmailTest.java

```diff
@@ -72,9 +73,16 @@ public void beforeTest() {
 				.requiredAction(UserModel.RequiredAction.UPDATE_EMAIL.name()).build();
 		prepareUser(user);
 		ApiUtil.createUserAndResetPasswordWithAdminClient(testRealm(), user, "password");
+		setRegistrationEmailAsUsername(testRealm(), false);
```

Comment:
Nitpick: Is it possible to move this to the end of new test method instead? See similar comment for `AbstractAppInitiatedActionUpdateEmailTest`.